### PR TITLE
update doc for kubelet option --reserved-cpus

### PR DESCRIPTION
**Pull request by jianzzha**
This website update is for the new kubelet CLI option: kubernetes/kubernetes#83592

### CODE REVIEW

**Comment by tengqm** [comment]
/lgtm
Defer to release maester for merge.

File: content/en/docs/tasks/administer-cluster/cpu-management-policies.md [hunk 1]
```diff
@@ -93,7 +93,10 @@ state file `cpu_manager_state` in the kubelet root directory.
 This policy manages a shared pool of CPUs that initially contains all CPUs in the
 node. The amount of exclusively allocatable CPUs is equal to the total
 number of CPUs in the node minus any CPU reservations by the kubelet `--kube-reserved` or
-`--system-reserved` options. CPUs reserved by these options are taken, in
+`--system-reserved` options. From 1.17, the CPU reservation list can be specified
+explictly by kubelet --reserved-cpus option. The explicit CPU list specified by
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/cpu-management-policies.md]: I recommend putting “--reserved-cpus” and other flags inside backticks.

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 2]
```diff
@@ -146,6 +146,28 @@ control group (`system.slice` on systemd machines for example).
...
+- **Kubelet Flag**: `--reserved-cpus=`
+
+`reserved-cpus` is meant to define an explict CPU set for OS system daemons and
+kubernetes system daemons. This option is added in 1.17 release. `reserved-cpus`
+is for systems that do not intent to define seperate top level cgroups for
+OS system daemons and kubernetes system daemons with regard to cpuset resource.
+If the Kubelet **does not** have `--system-reserved-cgroup` and `--kube-reserved-cgroup`,
+the explicit cpuset provided by `reserved-cpus` will take precedence over the CPUs
+defined by `--kube-reserved` and `--system-reserved` options.
+
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/reserve-compute-resources.md]: What goes after the `=` sign? I'd guess that I could specify, eg: `--reserved-cpus=8-15` but I'm not sure if I've guessed correctly.

Comment (jianzzha) [reply content/en/docs/tasks/administer-cluster/reserve-compute-resources.md]: That's right. Updated.

File: content/en/docs/tasks/administer-cluster/cpu-management-policies.md (hunk 1 above)
Comment (jianzzha) [reply content/en/docs/tasks/administer-cluster/cpu-management-policies.md]: thanks and updated.

**Pull request merged**